src/client/components/DisasterQuestion/index.js

```diff
@@ -0,0 +1,52 @@
+import Form from "react-bootstrap/Form";
+import PropTypes from "prop-types";
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+function DisasterQuestion(props) {
+  const { t } = useTranslation();
+
+  function onChange(event) {
+    props.onChange({
+      name: "recentDisasterChoice",
+      value: event.target.value,
+    });
+  }
+
+  const disasterDropDownChoices = [
+    t("retrocerts-certification.disaster-choices.choice-1"),
```

Comment:
I thought about doing this a fancier way, but I think it's nice to be able to copy and paste keys from the JSON file to find them in the codebase.

Reply:
That's true. You could have a comment with the full values and use a loop in the code. It does add to the download size for the user.

Reply:
Good point! It's getting late on the east coast, so I'm going to merge this and fix it Monday!